### PR TITLE
Update section on embed ipywidgets

### PR DESCRIPTION
**Pull request by guimillet**
In the current example, `single_widget` is only a class attribute, such that when creating a new instance of `MyComponent`, the same widget is displayed. The user rather would rather need to have a different instance of `single_widget` for each instance of `MyComponent`.

For that, I suggest to initialise the embedded ipywidget of `MyComponent` within the `__init__` function. This also gives the possibility of customising the embedded widget upon instantiating `MyComponent`, via keyword arguments given to the `_init__` function.

The patch also harmonises quote style used in the code example.

### CODE REVIEW

File: docs/template_usage.rst [hunk 1]
```diff
@@ -176,13 +176,16 @@ Other ipywidgets can be embedded by setting them in a trait and adding widget_se
...
         }]).tag(sync=True, **widgets.widget_serialization)
 
-        single_widget = traitlets.Any(
-            widgets.IntSlider(description='Single slider', value=40)
-        ).tag(sync=True, **widgets.widget_serialization)
+        single_widget = traitlets.Any().tag(sync=True, **widgets.widget_serialization)
+    
+        def __init__(self, **kwargs):
+            self.single_widget = widgets.IntSlider(description='Single slider', value=40)
+            self.single_widget.__init__(**kwargs)
```
Comment (maartenbreddels) [inline docs/template_usage.rst:183]: Calling the constructor twice cannot be right, it probably has weird side effects (creating 2 comms).
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40, **kwargs)
```

Comment (guimillet) [reply docs/template_usage.rst:183]: I agree, but the suggested change does not work when calling MyComponent(value=20), i.e. for arguments already defined.
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40)
            for k, v in kwargs.items():
                setattr(self.single_widget, k, v)
```
Alternatively, the default arguments are passed to `__init__`:
```
def __init__(self, description='Single slider', value=40, **kwargs):
    self.single_widget = widgets.IntSlider(
        description=description, 
        value=value, 
        **kwargs
    )
```
The second option may look more readable. Any preference?

File: docs/template_usage.rst [hunk 2]
```diff
@@ -155,7 +155,7 @@ A demonstration in a screen capture:
 Embed ipywidgets
 ----------------
 
-Other ipywidgets can be embedded by setting them in a trait and adding widget_serialization, accessing them in the with the `jupyter-widget` tag:
+Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the `jupyter-widget` tag:
```
Comment (12rambau) [inline docs/template_usage.rst:158]:
```suggestion
Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the ``jupyter-widget`` tag:
```

I think single quote are .md symbols, you need to use 2 of them in rst files.

Comment (guimillet) [reply docs/template_usage.rst]: Thanks for the suggestion, it is committed.

**Comment by mariobuikhuizen** [comment]
Thanks @guimillet!

**Pull request merged**